### ISSUE TITLE
Check contrast across dashboard

### ISSUE THREAD

**Issue by Bubballoo3** [OSC/ondemand#5169]
From #1001, there could be cases where contrast makes things difficult to read, and this could be an easy win for accessibility.

**Comment by Bubballoo3** [comment]
This should be nearly done after #5334. There are still a few widgets I can't seem to load on my dev dashboard (nsf_events, file_quotas, balances, xdmod), but I doubt there will be any issues there. We can close once we can double back on those last few widgets.

**Comment by moffatsadeghi** [comment]
@Bubballoo3 can this be closed out

**Comment by Bubballoo3** [comment]
Yes. I was able to review everything except the widgets, and have created #5370 to make sure we circle back to those at some point
